src/components/EtaxForm/EtaxForm.jsx

```diff
@@ -21,12 +23,23 @@ class EtaxForm extends React.Component {
     })
   }
 
+  submitForm = () => {
+    Api.post("/address", this.state)
+    .then(response => console.log(response))
+    .catch(error => console.log(error))
+  }
+
   render() {
     return (
-      <EqualLayoutForm
-        fields={ addressFields }
-        handleChange={ this.handleChange }
-      />
+      <div>
+        <EqualLayoutForm
+          fields={ addressFields }
+          handleChange={ this.handleChange }
+        />
+        <Button onClick={this.submitForm} variant="raised" color="primary">
```

Comment:
Minor Nitpick: Looks like there's a little bit of code style inconsistencies here in bracket spacing.

You could install ESLint to help you lookout for those, or use Prettier to automatically fix these code styling issues in your development tool of choice.